dockers/docker-orchagent/orchagent.sh

```diff
@@ -132,6 +132,15 @@ if [ ! -z "$HEARTBEAT_INTERVAL" ] && [ $HEARTBEAT_INTERVAL != "null" ]; then
     ORCHAGENT_ARGS+=" -I $HEARTBEAT_INTERVAL"
 fi
 
+# Enable SAI MACSec POST when:
+# - FIPS is enabled in SONiC (either in /proc/cmdline or /etc/fips/fips_enable); AND
+# - MACSec is enabled on platform.
+if grep -q "sonic_fips=1" /proc/cmdline || grep -q "1" /etc/fips/fips_enable ; then
+    if grep -q "macsec_enabled=1" /usr/share/sonic/platform/platform_env.conf 2>/dev/null ; then
```

Comment:
Can we use source platform_env.conf or something line this https://github.com/criteo-forks/sonic-buildimage/blob/a9caf1152e1ed3d91cf7adf390dd89dfeb8602c5/files/scripts/asic_status.sh#L5

Reply:
Hi @judyjoseph update review to source `platform_env.conf` as suggested.